### DISCUSSION TITLE
Acaba IBAX değeri ne olacak?

### DISCUSSION THREAD

**Discussion by pubgkralix01** [IBAX-io/go-ibax#132] [Ideas]
Acaba IBAX değeri ne olacak?

**Comment by pubgkralix01** [comment]
bir avax çıka bilir diye umuyorum belki daha iyisi.

**Reply by Grekny** [reply]
altyapısı gerçekten harika seneye bu zamanlarda avax ın yerine bu coini konuşuyor olabiliriz.
